tf-psa-crypto/tests/scripts/components-build-system.sh

```diff
@@ -22,3 +22,8 @@ component_test_cmake_tf_psa_crypto_out_of_source () {
     cd "$TF_PSA_CRYPTO_ROOT_DIR"
     rm -rf "$OUT_OF_SOURCE_DIR"
 }
+
+component_tf_psa_crypto_check_files () {
```

Comment:
This should rather be in a new components-basic-checks.sh file.

Reply:
Rebased and fixedx, thanks.